lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -1,18 +1,36 @@
 # -*- coding: utf-8 -*-
-
 # Copyright: Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
 
-# NOTE: this file is here to allow modules using the new attributes feature to
```

Comment:
How about keeping this note, now that the content is gone? (Same for other PR(s).)